include/CustomAssert.h

```diff
@@ -69,6 +69,6 @@ inline void _assert([[maybe_unused]] const char* file, [[maybe_unused]] const ch
 #define beammp_assert_not_reachable()                                                      \
     do {                                                                                   \
         beammp_errorf("Assertion failed in '{}:{}': Unreachable code reached. This may result in a crash or undefined state of the program.", __func__, _line); \
-        Sentry.LogAssert("code is unreachable", _file_basename, _line, __func__);          \
+        Sentry.LogAssert("code is unreachable", std::string(_file_basename).substr(std::string(_file_basename).find_last_of("/\\") + 1), _line, __func__);          \
```

Comment:
Not sure what the reasoning here is, please revert this change or explain your reasoning.

Reply:
Passing the variable directly to the function includes the full path of the file in the log message, making it harder to read and understand due to the presence of irrelevant information.

The whole point of this change is to produce log messages that are clearer and easier to understand, which can be a bit more helpful when trying to diagnose the issues in the server/code.

Reply:
Why in the excuse my language fuck would you change this, its not even in use?! Sorry but this is a waste of time.

Reply:
Plus, its ALREADY THE BASENAME. Its in the fucking name. Im done with this. Get out of our repo.